lib/exception_handling.rb

```diff
@@ -274,7 +274,8 @@ def send_exception_to_honeybadger(exception_info)
                                     error_message: exception.message.to_s,
                                     exception:     exception,
                                     context:       exception_info.honeybadger_context_data,
-                                    controller:    exception_info.controller_name)
+                                    controller:    exception_info.controller_name,
+                                    tags:          exception_info.honeybadger_tags.join(' '))
```

Comment:
Didn't their docs say comma-separated? I see you have a commit moving it to space. Is that a newer interface from them?

Reply:
Yeah, they said comma separated, but the actual behavior didn't support comma separated.

Reply:
Yes, I think a commend there and a link to their docs would be good.

Did you happen to try sending the array in to their interface? Maybe that was the sort of comma they were referring to?

Reply:
Found the PR where they began supporting space-delimited tags in addition to comma-separated for the ruby gem:
https://github.com/honeybadger-io/honeybadger-ruby/pull/422

Unfortunately, only comma-separated tags show up in official documentation:
https://docs.honeybadger.io/lib/javascript/guides/tagging-errors/